components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher/src/main/resources/publisher-api.yaml

```diff
@@ -2924,6 +2924,77 @@ paths:
           schema:
             $ref: '#/definitions/Error'
 
+  /threat-protection/policies:
+    get:
+      summary: Get All Threat Protection Policies
+      description: This can be used to get all defined threat protection policies
+      responses:
+        200:
+          description: Ok. List of policies is returned
+          schema:
+            $ref: '#/definitions/ThreatProtectionPolicyList'
+
+  /threat-protection/add-policy/{apiId}/{policyId}:
```

Comment:
URI does not seem to be in correct format.

Can you please explain what is the purpose of this API a bit? Summary says this is used to associate an existing policy with an API. Which API should we use to create a policy at the first time?

Reply:
Yeah, Policy creation API is in admin REST APIs because policies can only be created by an admin.